src/westpa/westext/weed/weed_driver.py

```diff
@@ -172,7 +178,8 @@ def prepare_new_iteration(self):
             for bin, newprob in zip(bins, binprobs):
                 bin.reweight(newprob)
 
-            weed_global_group.attrs['last_reweighting'] = n_iter
+            weed_global_group.resize((weed_global_group.shape[0] + 1), axis=0)
```

Comment:
The `last_reweighting` attribute is gone. Is that ok?

Reply:
Reinstated the last_reweighting attribute.